main/acle.md

```diff
@@ -9456,6 +9461,701 @@ are named after. All of the functions have external linkage.
   void *__arm_sc_memchr(void *s, int c, size_t n);
 ```
 
+## SVE2.1 functions
+
+The functions in this section are defined by the header file
+ [`<arm_sve.h>`](#arm_sve.h) when `__ARM_FEATURE_SVE2p1` is defined.
+
+Some instructions overlap with the SME and SME2 architecture extensions and
+are additionally available in Streaming SVE mode when __ARM_FEATURE_SME is
+non-zero or __ARM_FEATURE_SME2 are non-zero.
+For convenience, these variants are tagged with `[SVE2.1 or SME]` and
+`[SVE2.1 or SME2]` respectively.
+
+### SVE2.1 data-processing instructions.
+
+#### ADDQV, FADDQV
+
+Unsigned/FP add reduction of quadword vector segments.
```

Comment:
Addition is the same regardless of signed vs unsigned. Can we have a _s{16,32,64} variant?

Reply:
And also i think this needs _u8 and _s8?

Reply:
Hi George,
I am not sure we should add the signed types, the spec is specific writing it is for unsigned types
https://developer.arm.com/documentation/ddi0602/2023-09/SVE-Instructions/ADDQV--Unsigned-add-reduction-of-quadword-vector-segments-?lang=en
It cast all the types to unsigned.